### ISSUE TITLE
[Keboard Navigation - Help - JavaScript - Call a function]: All interactive controls are not accessible through keyboard on call a function pane.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2529]
**User Experience**
A disable user accustomed to using the keyboard for navigating through the UI elements on the system, will find it difficult to not be able to access all interactive controls on call a function pane via keyboard.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'JavaScript' control and select it.
5-Navigate to various controls lying on the pane opened.
6-Navigate to the pane opened and select any link listed under it.
7-Verify all interactive controls are accessible through keyboard or not on call a function pane.

**Actual Result**
Below interactive elements are not accessible through keyboard on call a function pane.
1.Open documentation in a new tab
2.Edit 
3.Run 

**Expected Result**
Below interactive elements are should accessible through keyboard on call a function pane.
1.Open documentation in a new tab
2.Edit 
3.Run 

**Note:** The same issue exists in scenario:
1. Help-->Projects-->Select any step like smiley buttons--> List File code, Download and Play button is not accessible via keyboard.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Suggested fix**
1. Always ensure each functionality of a page is keyboard accessible. 
 E.g. A page with images used as links changes when the user hovers over the image with a mouse. To provide keyboard users with a similar experience, the image is also changed when a user tabs to it 
2. Always try to use standard HTML form controls and link elements to provide keyboard operation and assistive technology interoperability of interactive user interface elements. 
3. Use of both keyboard-specific and mouse-specific events with code that has a scripting function associated with an event. 
Use mousedown with  keydown   
Use mouseup  with keyup   
Use click   with  keypress    
Use mouseover with  focus   
Use mouseout with blur   
 e.g. 
< img onclick="nextPage();" onkeypress="nextPage();" tabindex="0" src="arrow.gif"  
alt="Go to next page" >  

**Key References for Fix:** 
http://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html

**Please refer attachment file:**
[MAS2.1.1_Call a function_All interactive elements are not accessible through keyboard.zip](https://github.com/Microsoft/pxt/files/1176444/MAS2.1.1_Call.a.function_All.interactive.elements.are.not.accessible.through.keyboard.zip)
![mas2 1 1_call a function_interactive elements are not accessible through keyboard](https://user-images.githubusercontent.com/30459830/28619378-c00222f4-7225-11e7-8d0a-f38cb95e7712.png)
![mas2 1 1_projects_select any step_buttons are not accessible](https://user-images.githubusercontent.com/30459830/28712906-92861a5c-73aa-11e7-867a-e8a1e2e5ce7f.png)

**Comment by v-amik** [comment]
Review done

**Comment by veler** [comment]
The open documentation in new tab is now accessible [https://github.com/Microsoft/pxt/pull/2552](https://github.com/Microsoft/pxt/pull/2552). The other points are part of the documentation, I will work on it later, probably in a few days or weeks.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
